Tutorials/Multisite-Setup/index.md

```diff
@@ -0,0 +1,73 @@
+---
+versionFrom: 8.0.0
+meta.Title: "Multisite setup in Umbraco"
+product: "CMS"
+meta.Description: "A guide to multisite setup in Umbraco"
+---
+
+# Multisite setup
+
+This tutorial explains how to host multiple sites from one project/installation of Umbraco.
+For practical reasons, we recommend using [Baselines](../../Umbraco-Cloud/Getting-Started/Baselines/) on Umbraco Cloud projects.
+
+
+:::tip
+When using Baselines on Umbraco Cloud for a multisite solution, you would not need to worry about [Usage](../../Umbraco-Cloud/Setup-up/Usage/) limits, and could see better performance compared to having multiple websites in one project.
+:::
+
+If you are planning to create a multilingual site in Umbraco 8, please take a look at the [Multilanguage Setup](../Multilanguage-Setup/) tutorial instead.
+
+## Structuring your website
+
+The best way to handle a multisite solution is to have multiple root nodes in the Content section, where each root node would act as a separate website.
+
+Keep in mind all the websites in your solution will be using the same schema - meaning, in most cases, your content pages on website A will be using the same properties as on website B. 
+
+:::note
+If your site is hosted on Umbraco Cloud, you will need to map your sites' hostnames to the project.
+
+Before you can map your hostnames to individual websites in the solution, you should add them in the Hostnames page on the Cloud portal to ensure they are secured with TLS.
+
+![Adding hostnames to the project](images/1-addinghostnames.png)
+
+Keep in mind the [hostnames have to be configured in a specific way.](../../Umbraco-Cloud/Set-Up/Manage-Hostnames/).
+:::
+
+## Mapping the hostnames to individual websites/root nodes
+
+At this point you should have several root nodes, where each is a separate website. Once the hostnames are protected by TLS:
```

Comment:
The "Once the hostnames are protected by TLS" is very Cloud specific, right?
Would it make sense to simply leave it out? Or perhaps say something along these lines instead:
"Once you've configured DNS and hostnames follow these steps:"

What do you think?

```suggestion
At this point you should have several root nodes, where each is a separate website. Once the hostnames are protected by TLS:
```

Reply:
Done!